.goreleaser.yml

```diff
@@ -7,13 +7,20 @@ builds:
   goos:
   - windows
   - linux
-  - darwin 
+  - darwin
+  goarch:
+  - amd64
+  - arm
+  - arm64
+  goarm:
+  - 6
+  - 7
 archives:
 - format: tar.gz
   format_overrides:
   - goos: windows
     format: zip
-  name_template: "{{ .ProjectName  }}-{{ .Os  }}-{{ .Arch  }}"
+  name_template: "{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
```

Comment:
if we are to change the file names, then we can just remove this line and use the default

```suggestion
```